content/en/basics/the-blockchain/consensus/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Consensus"
 description: ""
-lead: ""
+lead: "In the Filecoin blockchain, network consensus is achieved using the Expected Consenus (EC) algorithm, a secret, fair and verifiable consensus protocol used by the network to agree on the overall chain state at each epoch, including the storage power currently available in the network."
```

Comment:
I'm not sure Expected Consensus needs to be capitalized.

Reply:
According to https://github.com/filecoin-project/consensus and https://spec.filecoin.io/algorithms/expected_consensus/, it looks like it should be capitalized

Reply:
Ah, ok then. Leave it as it is.